CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 - IntelliJ Plugin: downgrade Java to version 11.
 - Disable progress in mirrord-layer - can cause issues with forks and generally confusing now
   that agent is created by cli (and soon to be created by IDE plugin via cli).
+- mirrord-agent: Stealer feature changed from working per connection to now starting with
```

Comment:
I'd add a tl;dr that this is a background change that shouldn't have effect on users.